loaddb.py

```diff
@@ -194,16 +190,13 @@ def __init__(self, owner, filename, offset, time, xdict, processor):
     self.xdict = xdict
     self.processor = processor
 
-  def __cmp__(self, other):
-    ltime = self.time
-    rtime = other.time
-    # Descending time sort order, so that later dates go first.
-    if ltime > rtime:
-      return -1
-    elif ltime < rtime:
-      return 1
-    else:
-      return 0
+  def __lt__(self, other):
+    """Sorting only takes into account the time of a line."""
+    return self.time < other.time
```

Comment:
I think you got this backwards, based on the old comment. `cmp(x,y)` is negative when `x < y` and in this case that is when `self.time > other.time`

Reply:
Oops this should have been a reply but whatever.